meta-lmp-base/classes/lmp.bbclass

```diff
@@ -11,6 +12,23 @@ include conf/machine/include/lmp-partner-custom.inc
 # Allow customizations per factory level
 include conf/machine/include/lmp-factory-custom.inc
 
+provision_root_meta () {
+	if [ -n "${SOTA_TUF_ROOT_FETCHER}" ]; then
+		if [ -f "${SOTA_TUF_ROOT_FETCHER}" ] && [ -x "${SOTA_TUF_ROOT_FETCHER}" ]; then
+			bbplain "Fetching TUF root role metadata to the ostree-based rootfs..."
+
+			FACTORY="${LMP_DEVICE_FACTORY}" \
+			SOTA_TUF_ROOT_DIR="${SOTA_TUF_ROOT_DIR}" \
+			LOG_FILE="${SOTA_TUF_ROOT_LOG_FILE}" \
+				${SOTA_TUF_ROOT_FETCHER}
+		else
+			bbfatal "Provisioning of TUF root role metadata is turned ON, but the specified metadata fetcher is either doesn't exist or is not executable: ${TUF_ROOT_FETCHER}"
```

Comment:
Should be SOTA_TUF_ROOT_FETCHER instead of TUF_ROOT_FETCHER, right?

Reply:
Indeed, updated.